engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -318,6 +333,21 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     }
   }
 
+  private void addExtraJarsToClasspath(Set<String> jars)
+      throws InvocationTargetException, IllegalAccessException, MalformedURLException, NoSuchMethodException
+  {
+    List<URL> jarUrls = new LinkedList<>();
+    for (String jarPath : jars) {
+      File file = new File(jarPath);
+      URL url = file.toURI().toURL();
+      jarUrls.add(url);
+    }
+
+    ClassLoader prevCl = Thread.currentThread().getContextClassLoader();
+    ClassLoader urlCl = URLClassLoader.newInstance(jarUrls.toArray(new URL[jarUrls.size()]), prevCl);
+    Thread.currentThread().setContextClassLoader(urlCl);
```

Comment:
@tweise Is it expected that StramAppLauncher.runLocal() exposes different behavior compared to LocalMode.runApp()? I would expect StramAppLaunchar.runLocal to use LocalMode.runApp() so functionality is the same when application is launched in application test and using dtcli local mode.